content/en/docs/tasks/configure-pod-container/resize-container-resources.md

```diff
@@ -2,264 +2,192 @@
+* **Resource Removal:** Resource requests and limits cannot be entirely removed once set; they can only be changed to different values.
+* **Operating System:** Windows pods do not support in-place resize.
+* **Node Policies:** Pods managed by [static CPU or Memory manager policies](/docs/tasks/administer-cluster/cpu-management-policies/) cannot be resized in-place.
+* **Swap:** Pods utilizing [swap memory](/docs/concepts/architecture/nodes/#swap-memory) cannot resize memory requests unless the `resizePolicy` for memory is `RestartContainer`.
 
-- Only CPU and memory resources can be changed.
-- Pod QoS Class cannot change. This means that requests must continue to equal limits for Guaranteed
-  pods, Burstable pods cannot set requests and limits to be equal for both CPU & memory, and you
-  cannot add resource requirements to Best Effort pods.
-- Init containers and Ephemeral Containers cannot be resized.
-- Resource requests and limits cannot be removed once set.
-- A container's memory limit may not be reduced below its usage. If a request puts a container in
-  this state, the resize status will remain in `InProgress` until the desired memory limit becomes
-  feasible.
-- Windows pods cannot be resized.
+These restrictions might be relaxed in future Kubernetes versions.
 
+## Example 1: Resizing CPU without Restart
 
-## Create a pod with resource requests and limits
+First, create a Pod designed for in-place CPU resize and restart-required memory resize.
 
-You can create a Guaranteed or Burstable [Quality of Service](/docs/tasks/configure-pod-container/quality-service-pod/)
-class pod by specifying requests and/or limits for a pod's containers.
+{{% code_sample file="pods/resource/pod-resize.yaml" %}}
 
-Consider the following manifest for a Pod that has one Container.
+Create the pod:
 
-{{% code_sample file="pods/qos/qos-pod-5.yaml" %}}
+```shell
+kubectl create -f pod-resize.yaml
+```
 
-Create the pod in the `qos-example` namespace:
+This pod starts in the Guaranteed QoS class. Verify its initial state:
 
 ```shell
-kubectl create namespace qos-example
-kubectl create -f https://k8s.io/examples/pods/qos/qos-pod-5.yaml
+# Wait a moment for the pod to be running
+kubectl get pod resize-demo --output=yaml
 ```
 
-This pod is classified as a Guaranteed QoS class requesting 700m CPU and 200Mi
-memory.
+Observe the `spec.containers[0].resources` and `status.containerStatuses[0].resources`. They should match the manifest (700m CPU, 200Mi memory). Note the `status.containerStatuses[0].restartCount` (should be 0).
 
-View detailed information about the pod:
+Now, let's increase the CPU request and limit to `800m`. We use `kubectl patch` with the `--subresource resize` flag.
 
 ```shell
-kubectl get pod qos-demo-5 --output=yaml --namespace=qos-example
+kubectl patch pod resize-demo --subresource resize --patch \
```

Comment:
What namespace is change happening in? Do you need to create a namespace first?

Reply:
(see above)